api/users/serializers.py

```diff
@@ -18,7 +19,12 @@ class UserSerializer(JSONAPISerializer):
     family_name = ser.CharField(required=False, allow_blank=True, help_text='For bibliographic citations')
     suffix = ser.CharField(required=False, allow_blank=True, help_text='For bibliographic citations')
     date_registered = ser.DateTimeField(read_only=True)
-    gravatar_url = ser.URLField(required=False, read_only=True, help_text='URL for the icon used to identify the user. Relies on http://gravatar.com ')
+
+    profile_image_url = ser.SerializerMethodField(required=False, read_only=True)
```

Comment:
`required` and `read_only` are unnecessary, since `SerializerMethodFields` are already read-only.